PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Submitting a Pull Request
+
+* Fork the repository by clicking the fork button on top right corner of the page
+* Clone the target repository. To clone, click on the clone button and copy the https address. Then run
+
+`git clone [HTTPS-ADDRESS]`
+
+* Go to the cloned directory by running
+
+`cd [NAME-OF-REPO]`
+
```

Comment:
PR template should not have the method to create a PR, rather fields to fill while sending PR.

Reply:
ff492c90efc51e2839af7cbe0a4ecd7ae1bb141e fixes the pr template